src/components/vouchers/VoucherRedeem.tsx

```diff
@@ -84,6 +98,9 @@ export const VoucherRedeem = ({ quota, voucherId }: { quota: TQuota; voucherId:
         }
       });
 
+      setIsRedeemedVoucher(true);
+      setIsRedeemingVoucher(true);
```

Comment:
can we use `finally` for these tow to not duplicate them in `try` and `catch`?

Reply:
Yeah, you're absolutely right. 🤩